src/Identity/test/Identity.FunctionalTests/Testing.DefaultWebSite.StaticWebAssets.V4.xml

```diff
@@ -1,3 +1,3 @@
 <StaticWebAssets Version="1.0">
```

Comment:
Nit: should this filename have "BootstrapV4" instead of just "V4"? Same for the other one.

Reply:
We can table this for now since these have no effect as they will be used for the functional tests that don't exist currently, the next PR we can rename these, there used to be V3/V4 versions of these, so this was part of the changes that were undone